packages/avatar/react.d.ts

```diff
@@ -0,0 +1,23 @@
+import {
```

Comment:
cp and adjust of dist/esm/index.ts

should probably create a script to do the copy and adjust in a yarn build

Reply:
I concur on this